### PR TITLE
Explicit enumeration

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: bin/runtime/src/lib.rs [hunk 1]
```diff
@@ -859,59 +859,59 @@ impl pallet_proxy::Config for Runtime {
...
-        Vesting: pallet_vesting,
-        Utility: pallet_utility,
-        Multisig: pallet_multisig,
-        Sudo: pallet_sudo,
-        Contracts: pallet_contracts,
-        NominationPools: pallet_nomination_pools,
-        Identity: pallet_identity,
-        CommitteeManagement: pallet_committee_management,
-        Proxy: pallet_proxy,
+        System: frame_system = 1,
```
Comment (DamianStraszak) [inline bin/runtime/src/lib.rs:862]: I think it would be great to run subwasm on this change. I suspect it's 0-based by default :P

Comment (pmikolajczyk41) [reply bin/runtime/src/lib.rs:862]: @kostekIV have you run subxt for standard runtime as well? if so (no changes), then it might be 1-based indeed...

Comment (kostekIV) [reply bin/runtime/src/lib.rs]: nope, of course its from 0. Fixed

**Comment by pmikolajczyk41** [comment]
1. have you regenerate standard subxt runtime?
2. shouldn't we have some gap after removing randomness pallet?

**Review by DamianStraszak — APPROVED** [review]

**Comment by kostekIV** [comment]
> 1. have you regenerate standard subxt runtime?
> 
>     2. shouldn't we have some gap after removing randomness pallet?

1. yep, now I have pushed it. Some minor changes most likely not related to this pr.
2. Nope, we havent yet deleted it. Its planned for after the release 13

**Pull request merged**